storage/index.js

```diff
@@ -1 +1,15 @@
 // JavaScript source code
+var server = require('./app.js');
+var storageModule;
+var config = server.config;
+var client;
+
+// Use the following to require different types of DB
+if (config.db.type =='mongo'){
+    storageModule = require('./mongo/index.js');
+    client = new storageModule(config);
+}
+else
+    storageModule = require('sql');
+
+client.mongo.init();
```

Comment:
Storage is module for us. The definition of which is present at storage/index.js so module.exports is missing and instead of using client.mongo.init() and init function should be used which would be exporting storage.init